httpx/_dispatch/asgi.py

```diff
@@ -1,8 +1,8 @@
-import typing
+from typing import Callable, List, Tuple
```

Comment:
Are you now nudged towards switching to that style too? I've been thinking about it lately. :-)

Reply:
Happy enough either way, but yeah it works nicer in more verbose cases like the httpcore API.

Ie I'd rater we've got this kind of thing...

```python
    async def request(
        self,
        method: bytes,
        url: Tuple[bytes, bytes, int, bytes],
        headers: List[Tuple[bytes, bytes]] = None,
        stream: ContentStream = None,
        timeout: Dict[Optional[float]] = None,
    ) -> Tuple[bytes, int, bytes, List[Tuple[bytes, bytes]], ContentStream]:
```

Than this:

```python
    async def request(
        self,
        method: bytes,
        url: typing.Tuple[bytes, bytes, int, bytes],
        headers: typing.List[typing.Tuple[bytes, bytes]] = None,
        stream: ContentStream = None,
        timeout: typing.Dict[typing.Optional[float]] = None,
    ) -> Tuple[bytes, int, bytes, typing.List[typing.Tuple[bytes, bytes]], ContentStream]:
```